### PR TITLE
Bugfix/776 contingent event. Fixes #776.

### LINKED ISSUE #776: Contingent Event should not have restrictions on planned datetimes
The equivalence axiom on `ContingentEvent` is:

```
gist:ContingentEvent
	a owl:Class ;
	owl:equivalentClass [
		a owl:Class ;
		owl:intersectionOf (
			gist:Event
			[
				a owl:Restriction ;
				owl:onProperty gist:hasMagnitude ;
				owl:someValuesFrom gist:Percentage ;
			]
			[
				a owl:Restriction ;
				owl:onProperty gist:plannedEndDateTime ;
				owl:someValuesFrom xsd:dateTime ;
			]
			[
				a owl:Restriction ;
				owl:onProperty gist:plannedStartDateTime ;
				owl:someValuesFrom xsd:dateTime ;
			]
		) ;
	] ;
```
If the event which the contingent event is dependent on does not have planned dates, the contingent event cannot have planned dates. One might specify some time interval after the primary event's date, but not actual dates. 

We should consider the additional restriction also used on `ContingentObligation`:

```
rdfs:subClassOf
[
    a owl:Restriction ;
    owl:onProperty gist:isTriggeredBy ;
    owl:someValuesFrom gist:Event ;
] ;
```

### PR DESCRIPTION
**Pull request by coltonglasgow**
This PR removed dateTime restrictions from `gist:ContingentEvent` and adds a `skos:example`. 

See the issue here: https://github.com/semanticarts/gist/issues/776

### CODE REVIEW

**Review by rjyounes — CHANGES_REQUESTED** [review]
The gist changes look good. You will need to add a release note. The process has changed from what's in Contributing.md (I'm updating that on another branch): you put the note in a separate file. You'll see how this is done by looking at some of the other PRs. Let me know if you have questions.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Hi Colton - just a couple modifications needed to the release note and we're good to go.

**Comment by rjyounes** [comment]
@coltonglasgow I just saw Dave's comment: I think the canonical example and what ought to be in the definition is the event that triggers some indemnity. The death benefit payout on a life insurance policy is the death of a very specific person. Fire insurance is contingent on a particular building burning down.

Please incorporate this into the definition.

File: gistCore.ttl [hunk 1]
```diff
@@ -472,17 +472,17 @@ gist:ContingentEvent
...
+				owl:onProperty gist:isTriggeredBy ;
+				owl:someValuesFrom gist:Event ;
 			]
 		) ;
 	] ;
 	skos:definition "An event with a probability of happening in the future, and usually dependent upon some other event or condition."^^xsd:string ;
+	skos:example
+		"Fire insurance is contingent on a particular building burning down"^^xsd:string ,
+		"Sell 20 shares of stock in a given company when the price drops below $200/share."^^xsd:string ,
+		"The death benefit payout on a life insurance policy following the death of a very specific person."^^xsd:string
```
Comment (coltonglasgow) [inline gistCore.ttl:484]: @rjyounes, I changed `is` to `following` in this example. Let me know if you want me to change it.

File: gistCore.ttl [hunk 2]
```diff
@@ -472,17 +472,17 @@ gist:ContingentEvent
...
-				owl:someValuesFrom xsd:dateTime ;
+				owl:onProperty gist:isTriggeredBy ;
+				owl:someValuesFrom gist:Event ;
 			]
 		) ;
 	] ;
 	skos:definition "An event with a probability of happening in the future, and usually dependent upon some other event or condition."^^xsd:string ;
+	skos:example
+		"Fire insurance is contingent on a particular building burning down"^^xsd:string ,
+		"Sell 20 shares of stock in a given company when the price drops below $200/share."^^xsd:string ,
```
Comment (coltonglasgow) [inline gistCore.ttl:483]: @rjyounes, I kept this example, I believe it is still relevant.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Just one more small tweak.

**Review by rjyounes — CHANGES_REQUESTED** [review]
Just one more small tweak.

**Review by rjyounes — APPROVED** [review]
Yay!

**Pull request merged**